helm_deploy/values-prod.yaml

```diff
@@ -56,6 +56,7 @@ env:
   COMPLEXITY_OF_NEED_URI: https://complexity-of-need.hmpps.service.justice.gov.uk
   PRISONS_WITH_OFFENDERS_THAT_HAVE_COMPLEX_NEEDS: "AGI,BZI,DHI,DWI,ESI,EWI,FHI,LNI,NHI,PFI,SDI,STI"
   PRISONS_WITH_MANAGE_ADJUDICATIONS_ENABLED: "RNI,WLI"
+  PRISONS_FOR_MANAGE_PRISON_BOOKINGS: "HEI"
```

Comment:
this is slightly confusing - you're enabling it for `HEI` but haven't provided the url so I'm guessing it is disabled

Reply:
`MANAGE_PRISON_VISITS_URL: https://manage-prison-visits.prison.service.justice.gov.uk/` 😕

Reply:
Already in there so marking as resolved.